packages/desktop-client/src/components/settings/Experimental.tsx

```diff
@@ -116,6 +116,12 @@ export function ExperimentalFeatures() {
             >
               <Trans>Rule action templating</Trans>
             </FeatureToggle>
+            <FeatureToggle
+              flag="upcomingLengthAdjustment"
+              // feedbackLink="https://github.com/actualbudget/actual/issues/3606" TODO: Create feedback issue
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏created it for you :)
```suggestion
              feedbackLink="https://github.com/actualbudget/actual/issues/3660"
```